packages/components/molecules/f-searchbox/src/components/formElements/FormFullAddressSearchSuggestions.vue

```diff
@@ -264,18 +310,19 @@ export default {
     background: $white;
     border-bottom-left-radius: $shell-border-radius;
     border-bottom-right-radius: $shell-border-radius;
-    position: absolute;
-    left: 0;
-    right: 0;
-    top: #{$shell-top-offset}px;
     box-sizing: border-box;
     overflow-y: scroll;
     z-index: 1000;
+    margin-top: 5px;
     height: calc(100vh - #{$shell-top-offset}px);
 
-    @include media('>=784px') {
+    @include media('>767px') {
```

Comment:
Can we use `@include media('>mid')` here? mid = 768px

Reply:
Fixed.